### PR TITLE
Big renaming

### PR DESCRIPTION
**Pull request by fpagnoux**
Code ongoing, not for review yet.

### CODE REVIEW

**Comment by laem** [comment]
New names are way better :sweat_smile:

**Comment by MattiSG** [comment]
:dance: :heart: 💯

**Comment by fpagnoux** [comment]
Summary of the renaming :
=====================

Introduced variables:
----------------------------
`invalidite` (different concept than `handicap`)

Deprecated variables:
-----------------------------

* `af_enfant_a_charge` (duplicate of `prestations_familiales_enfant_a_charge`)
* `asf_enfant` (merged in `asf`)
* `isol` (just the negation of  `en_couple`)
* `pfam_ressources_i` (merged in `prestations_familiales_base_ressources`)
* `rmi_nbp` (just need to sum `nb_parents` and `rsa_nb_enfants`)
* `sal_pen_net` (merged in `traitements_salaires_pensions_rentes`)

Renaming:
--------------

* `abat_sal_pen` -> `abattement_salaires_pensions`
* `af_forf` -> `af_allocation_forfaitaire`
* `af_forf_complement_degressif` -> `af_allocation_forfaitaire_complement_degressif`
* `af_forf_nbenf` -> `af_allocation_forfaitaire_nb_enfants`
* `af_forf_taux_modulation` -> `af_allocation_forfaitaire_taux_modulation`
* `af_majo` -> `af_majoration`
* `al_pac` -> `al_nb_personnes_a_charge`
* `als_nonet` -> `als_non_etudiant`
* `alset` -> `als_etudiant`
* `ape_temp` -> `ape_avant_cumul`
* `apje_temp` -> `apje_avant_cumul`
* `asi_elig` -> `asi_eligibilite`
* `aspa_elig` -> `aspa_eligibilite`
* `biact` -> `biactivite`
* `birth` -> `date_naissance`
* `br_mv` -> `asi_aspa_base_ressources`
* `br_mv_i` -> `asi_aspa_base_ressources_i`
* `br_pf` -> `prestations_familiales_base_ressources`
* `br_pf_i` -> `prestations_familiales_base_ressources_i`
* `categ_inv` -> `aeeh_niveau_handicap`
* `concub` -> `en_couple`
* `invalide` -> `handicap`
* `nb_enfant_rsa` -> `rsa_nb_enfants`
* `nb_par` -> `nb_parents`
* `pen_net` -> `revenu_assimile_pension_apres_abattements`
* `pfam_enfant_a_charge` -> `prestations_familiales_enfant_a_charge`
* `rev_act_nonsal` -> `revenu_activite_non_salariee`
* `rev_act_sal` -> `revenu_activite_salariee`
* `rev_pen` -> `revenu_assimile_pension`
* `rev_sal` -> `revenu_assimile_salaire`
* `rto_declarant1` -> `retraite_titre_onereux_declarant1`
* `rto_net` -> `retraite_titre_onereux_net`
* `rto_net_declarant1` -> `retraite_titre_onereux_net_declarant1`
* `salcho_imp` -> `revenu_assimile_salaire_apres_abattements`
* `smic55` -> `autonomie_financiere`
* `statut_occupation` -> `statut_occupation_logement`
* `statut_occupation_famille` -> `statut_occupation_logement_famille`
* `statut_occupation_individu` -> `statut_occupation_logement_i`
* `tns_employe` -> `tns_avec_employe`
* `tspr` -> `traitements_salaires_pensions_rentes`
* `type_sal` -> `categorie_salarie`

**Comment by laem** [comment]
A petit double quote en trop dans le dernier commit :grimacing: 

Changelog :+1:

**Comment by benjello** [comment]
@fpagnoux : super boulot !

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,8 +1,51 @@
 # Changelog
 
-## 0.5.5.dev0 - next release
-
-* TODO Fill this changes list while developing
+## 1.0
```
Comment (cbenz) [inline CHANGELOG.md]: :+1:

**Comment by fpagnoux** [comment]
I think this is a good opportunity to set some common guidelines about openfisca variables naming.
I put what I gathered while going through the code [here](https://github.com/openfisca/openfisca-france/blob/renaming/NAMING_GUIDELINES.md).
Once we reach a consensus, I'll move it into the doc, but now feel free to comment on the code.

**Comment by cbenz** [comment]
@fpagnoux Very nice! I really like the fact to assume acronyms when they are publicly/officialy known.
Perhaps time to start a "legislation glossary" in the doc?

**Comment by fpagnoux** [comment]
Related to this topic: the need for prefixes seems to show that we are reaching some limits of the model where there is only one global namespace. See [here](https://github.com/sgmap/openfisca-france/commit/7dd19c212cb57f1b7e9de302d1a0799059492e66#commitcomment-17117794) the related discussion.

File: NAMING_GUIDELINES.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+Not specifying the scope of a specific variable should be avoided, as it is confusing for other users.
+>**OK**: ir_nb_pac
+
+>**KO**: nb_pac
+
+### Entity suffixes
+
+It happens that several variables have the same meaning, but for different entitities (individus, familles, etc.). Standard suffixes should be used to distinguish them.
+>**OK**: ass_base_ressources_i, statut_occupation_logement_famille
```
Comment (laem) [inline NAMING_GUIDELINES.md]: So `i` in `ass_base_ressources_i` stands for `individu` right ? Why abbreviate it ? The second is clearer.
Link to http://doc.openfisca.fr/key-concepts.html#persons-entities-and-roles ?

Comment (fpagnoux) [inline NAMING_GUIDELINES.md]: 👍 
Good remark, I didn't push the logic till the end.

**Comment by fpagnoux** [comment]
I'm done with renaming, is everyone ok with the list proposed here (and the guidelines) ? If you have another renaming to suggest, feel free to commit on the branch. 
If no ajustement is needed, I will:
- [ ] Add the guidelines to the wiki, and remove the file from the repo.
- [ ] Publish a release before renaming (`0.5.5`)
- [ ] Publish a release after renaming (`1.0`)

**Pull request merged**